website/app/routes/show.js

```diff
@@ -108,6 +108,7 @@ export default class ShowRoute extends Route {
 
         // check if there are special needs for the page layout
         const hasCover = frontmatter?.layout?.cover ?? true;
+        const hasTabs = frontmatter?.layout?.tabs ?? true;
```

Comment:
I agree it would be cleaner if the solution didn't involve making frontmatter changes. I was trying to follow the pattern used for checking if things like the cover or sidecar were present, which relies on those things being declared in frontmatter. 

I found a solution that doesn't rely on frontmatter that I think makes it a lot cleaner. Just updated the PR.

Reply:
@dchyun can you update the PR description accordingly? thanks

Reply:
Updated!